sdk/src/main/scala/com/horizen/SidechainApp.scala

```diff
@@ -112,6 +112,9 @@ class SidechainApp @Inject()
   val calculatedSysDataConstant: Array[Byte] = CryptoLibProvider.sigProofThresholdCircuitFunctions.generateSysDataConstant(signersPublicKeys.map(_.bytes()).asJava, sidechainSettings.withdrawalEpochCertificateSettings.signersThreshold)
   log.info(s"calculated sysDataConstant is: ${BytesUtils.toHexString(calculatedSysDataConstant)}")
 
+  val forgerList: Seq[(PublicKey25519Proposition, VrfPublicKey)] = sidechainSettings.forger.forgerList.map(el =>
```

Comment:
According to the comments in the `SidechainSettings` class, here we may easily map `ForgerKeysData` entries to
`Seq[(PublicKey25519Proposition, VrfPublicKey)]` structure.
Note: use proper serializers for parsing bytes to `PublicKey25519Proposition / VrfPublicKey`

Reply:
Done